lms/djangoapps/instructor/views/data_access_constants.py

```diff
@@ -16,6 +16,21 @@ def __init__(self, query_type, inclusion, entity_id, filtering, entity_name):
         self.entity_name = entity_name
 
 
+class QueryOrigin:
```

Comment:
readability and also universal assignment so if we need to add it/change it anywhere it's easy